wgpu-core/src/binding_model.rs

```diff
@@ -221,12 +221,13 @@ pub enum BindingZone {
 }
 
 #[derive(Clone, Debug, Error)]
-#[error("Too many bindings of type {kind:?} in {zone}, limit is {limit}, count was {count}")]
+#[error("Too many bindings of type {kind:?} in {zone}, limit is {limit}, count was {count}. Check the required limit {config_name} used on request_device")]
 pub struct BindingTypeMaxCountError {
     pub kind: BindingTypeMaxCountErrorKind,
     pub zone: BindingZone,
     pub limit: u32,
     pub count: u32,
+    pub config_name: &'static str,
```

Comment:
Instead of this being a separate string/argument, could this be a `to_config_str()` or so on `BindingTypeMaxCountErrorKind`, which matches on what it is, and prints the name

Reply:
Changed :)